scripts/run_performance_tests.sh

```diff
@@ -76,7 +76,17 @@ chmod 744 $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
 # Wait for the servers to come up.
 while ! nc -vz localhost 9501; do sleep 1; done
 
-$PYTHON_CMD scripts/backend_tests.py --test_path='core/tests/performance_tests' $@ 
+# Install xvfb if not on travis, Used in frontend, e2e tests and performance tests.
+if [ "$TRAVIS" = true ]; then
+  export XVFB_PREFIX=""
+else
+  # This installs xvfb for systems with apt-get installer like Ubuntu, and will fail for other systems.
+  # TODO: Install/provide xvfb for other systems.
```

Comment:
Please use the syntax "TODO(gvishal)" for all todos you write. The name in brackets is the person who wrote the to-do; this is useful in the future if clarification of the todo is needed.